### DISCUSSION TITLE
Alerting: show alert state indicator on a panel for Grafana 8 alerts

### DISCUSSION THREAD

**Discussion by domasx2** [grafana/grafana#38458] [Alerting]
Legacy alerts used to show a green/red heart indicator in dashboard panel header based on alert state.

This currently is not supported for Grafana 8 alerts, but I think it could be made to work. Needs some investigation.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
